### DISCUSSION TITLE
The switch is configured with a dual network port trunk

### DISCUSSION THREAD

**Discussion by QinQiang666** [cobbler/cobbler#2616] [Q&A]
The switch is configured with a dual network port trunk ，data transmission problems in the process of PXE. Now the solution is to pull out a network cable or drop a network port on the side of the switch to solve the problem。

Is there any other way to solve it？

**Comment by SchoolGuy** [comment]
Is your Cobbler system defined inside Cobbler with both interfaces? But in general I don't have an idea ad-hoc.

The only thing I found which could be helpful is the following blog article: <http://www.vreference.com/2011/05/03/how-to-pxe-boot-from-your-trunked-vmnic0/>
